app/modules/submissions/actions/submission-clone.js

```diff
@@ -13,11 +15,11 @@ import { getClonePath } from "../../../lib/pathutils"
 
 export function submissionCloneFunc (clone) {
   return (submissionProps) => {
-    return (dispatch, getState) => {
+    return async (dispatch, getState) => {
       const submissionsBaseDirectory = cloneDestination(getState())
       const assignmentName = name(getState())
       const submissionAuthorUsername = submissionProps.username
-
+      const accessToken = await keytar.findPassword("Classroom-Desktop")
```

Comment:
Fetch access token from Node keytar, if not found returns null

Reply:
Any thoughts on handling the `null` scenario here (imagine a scenario where the user goes and removes the token from the keychain)?

Reply:
Right now if the token comes back as null, it will just try to clone with no token and if the repository is private then it will say that the submission clone failed. We might want to add a better message or ask for reauthorization in the future. Added documentation to clarify the approach here, I think for this PR this functionality is fine.